tests/plotting/test_plotting.py

```diff
@@ -244,6 +245,20 @@ def test_pbr(sphere):
     pl.show(before_close_callback=verify_cache_image)
 
 
+@skip_not_vtk9
+@skip_windows
+@skip_mac
+@pytest.mark.skipif(CI_WINDOWS, reason="Windows CI testing segfaults on pbr")
```

Comment:
Does it make sense to have both `@skip_windows` and a manual skipif on `CI_WINDOWS`? I could imagine that CI_WINDOWS would be more informative due to the segfault message, but shouldn't that decorator come first then? Not sure.

Reply:
```suggestion
@skip_windows
@skip_mac
```

We really only need one skip windows.